docs/ref/templates/builtins.txt

```diff
@@ -903,6 +903,25 @@ attribute and calling the result ``country_list``.
 * ``list`` -- a list of all items in this group (e.g., a list of all cities
   with country='India').
 
+.. versionchanged:: 1.11
```

Comment:
If you want this example to remain after the versionchanged annotation is removed, you should make it self-contained as described in https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features.